build.gradle

```diff
@@ -393,7 +393,7 @@ dependencies {
 
     testCompile group: 'com.h2database', name: 'h2'
     testCompile "com.github.tomakehurst:wiremock:2.19.0"
-    testCompile group: 'org.mockito', name: 'mockito-core', version: '3.6.28'
+    testCompile group: 'org.mockito', name: 'mockito-core', version: '3.4.6'
```

Comment:
any reason for downgrade ?

Reply:
All junit fails, we need to analyse more on this